internal/view/app.go

```diff
@@ -63,38 +137,79 @@ func (a *App) Init(ctx context.Context, profiles, regions []string, version stri
 		"region":  a.region(),
 	}
 	a.Views()["info"] = ui.NewInfo(infoData)
+	a.toggleHeader(true)
 
-	if err := a.Content.Init(ctx); err != nil {
-		return err
-	}
-	a.Content.Stack.AddListener(a.Menu())
-	a.Content.Stack.AddListener(a.Crumbs())
-	a.App.Init()
-	a.SetInputCapture(a.keyboard)
-	a.bindKeys()
+}
+
+func (a *App) handleGCP() {
+	ctx := a.GetContext()
+	ctx = context.WithValue(ctx, internal.KeySelectedCloud, "GCP")
+	ctx = context.WithValue(ctx, internal.KeyActiveProject, "centering-aegis-400910")
+	a.SetContext(ctx)
 
-	a.command = NewCommand(a)
-	if err := a.command.Init(); err != nil {
-		return err
-	}
-	a.CmdBuff().SetSuggestionFn(a.suggestCommand())
-	a.layout(ctx)
-	return nil
 }
 
+func (a *App) showCloudSelectionScreen() {
+	cloudSelectionTable := ui.NewTable("Select Cloud")
+	cloudSelectionTable.Init(a.context)
+
+	cloudSelectionTable.SetSelectedFunc(func(row int, column int) {
+		if row == 0 {
+			return
+		}
+		seletedCloud := availableCloud[row-1]
+		switch seletedCloud {
+		case "AWS":
+			a.handleAWS()
+			if err := a.command.defaultCmd(); err != nil {
+				panic(err)
+			}
+			a.Main.SwitchToPage(internal.AWS_SCREEN)
+		case "GCP":
+			a.handleGCP()
+			if err := a.command.defaultCmd(); err != nil {
```

Comment:
Seems like in any case need to call `defaultCmd`.  I think we can extract this common part out to switch block, what do you think?